qiita_db/analysis.py

```diff
@@ -165,6 +169,71 @@ def create(cls, owner, name, description, parent=None, from_default=False):
         a_id = conn_handler.execute_queue(queue)[0]
         return cls(a_id)
 
+    @classmethod
+    def delete(cls, _id):
```

Comment:
Thanks @antgonza just one minor thing: can you add a comment with a reference to the issue? Like `# TODO: issue #1176`, so we have a pointer in the code and we don't forget about it.